source/includes/gcp/_forwarding_rules.md

```diff
@@ -1 +1,15 @@
-#### Forwarding rules
\ No newline at end of file
+#### Forwarding rules
+
+<!-------------------- DELETE A FORWARDING RULE -------------------->
+
+#### Delete a forwarding rule
+
+```shell
+curl -X DELETE \
+   -H "MC-Api-Key: your_api_key" \
+   "https://cloudmc_endpoint/v1/services/gcp/test-area/forwardingrules/5459493658413662148"
+```
+
+<code>DELETE /services/<a href="#administration-service-connections">:service_code</a>/<a href="#administration-environments">:environment_name</a>/forwardingrules/:id</code>
+
+Destroy an existing global forwaring rule.
```

Comment:
```suggestion
Delete an existing global forwarding rule.
```

Reply:
I have checked other delete docs, it always use destroy, so i am not very sure which word should i use

Reply:
@xrrlzzq if everywhere else uses destroy stick to it :)

Reply:
Ok,i will changed back and then merge, Thanks